<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>linearOptions</class>
 <widget class="QDialog" name="linearOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pictures/appIcon.png</normaloff>../pictures/appIcon.png</iconset>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>300</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="DirectContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#DirectContainer{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.954545, stop:0.471591 rgba(151, 255, 112, 255), stop:0.965909 rgba(227, 255, 207, 255));
}</string>
   </property>
   <widget class="QLabel" name="directLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 30pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>Direct methods</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="GaussEliminationButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#GaussEliminationButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
}

#GaussEliminationButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Gauss Elimination</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GaussJordanButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#GaussJordanButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
}

#GaussJordanButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Gauss-Jordan</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LUcomboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#LUcomboBox {
  transform: translate(-50%, -50%);
}
#LUcomboBox select {
  background-color: #0563af;
  color: black;
  padding: 12px;
  width: 250px;
  border: none;
  font-size: 15px;
  box-shadow: 0 5px 25px rgba(0, 0, 0, 0.2);
  -webkit-appearance: button;
  appearance: button;
  outline: none;
}

#LUcomboBox::before {
  content: &quot;\f13a&quot;;
  font-family: FontAwesome;
  position: absolute;
  top: 0;
  right: 0;
  width: 20%;
  height: 100%;
  text-align: center;
  font-size: 28px;
  line-height: 45px;
  color: black;
  background-color: rgba(255, 255, 255);
  pointer-events: none;
}

#LUcomboBox:hover::before {
  color: black;
  background-color: rgba(255, 255, 255,0.5);
}

#LUcomboBox select option {
  padding: 30px;
}</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <item>
     <property name="text">
      <string>Doolittle Form</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Crout Form</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cholesky Form</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="LUButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#LUButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
}

#LUButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>LU Decomposition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#previousButton{
	border: 1.3px solid #ccc;
	border-radius:5px; 
	font: 9pt &quot;MS Shell Dlg 2&quot;;
	
	background-color: rgb(90, 90, 90);
}

#previousButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Previous page</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../pictures/backArrow.png</normaloff>../pictures/backArrow.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>871</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.778409, y2:0.818, stop:0.528409 rgba(247, 255, 149, 255), stop:0.926136 rgba(114, 163, 255, 255));</string>
   </property>
   <widget class="QPushButton" name="JacobiButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#JacobiButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
}

#JacobiButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Jacobi-Iteration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeidelButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#SeidelButton{
	border: 1.3px solid #ccc;
	border-radius:15px; 
	background-color:#624587;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
}

#SeidelButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Gauss-Seidel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 30pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:5px</string>
    </property>
    <property name="text">
     <string>In-Direct methods</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
